lib/util.js

```diff
@@ -32,6 +33,9 @@ exports.asUserJson = function (user) {
     userJson.roles.unshift('USER')
   }
 
+  userJson.authorisations = {
+    patreon: user.identities.filter((u) => u.provider === 'patreon').length > 0
```

Comment:
This will error if the `identities` field is not defined, which was happening to me locally.